modules/@apostrophecms/asset/lib/webpack.config.js

```diff
@@ -1,9 +1,12 @@
 const path = require('path');
-
 const merge = require('webpack-merge');
-
 const scss = require('./webpack.scss');
 const vue = require('./webpack.vue');
+let BundleAnalyzerPlugin;
+
+if (process.env.APOS_BUNDLE_ANALYZER) {
+  BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
Should we make a note to remove this at some point? Should it live on in the eventual stable release?